package.json

```diff
@@ -25,7 +25,7 @@
     "@justeat/f-icons": "^0.3.4",
     "@justeat/f-toggle": "^0.11.2",
     "@justeat/f-validate": "^0.14.0",
-    "@justeat/fozzie": "^0.33.0",
+    "@justeat/fozzie": "^0.36.0",
```

Comment:
Can you bump the version number in the `package.json` too (I know we don't release it as a package iteself but it helps to keep it in sync and to have a version history)